controllers/token.go

```diff
@@ -172,6 +172,7 @@ func (c *ApiController) GetOAuthCode() {
 // @router /login/oauth/access_token [post]
 func (c *ApiController) GetOAuthToken() {
 	grantType := c.Input().Get("grant_type")
+	refreshToken := c.Input().Get("refresh_token")
```

Comment:
Why pass the token as parameter?

Reply:
What's wrong with it? The current refresh token endpoint is the same
https://github.com/casdoor/casdoor/blob/master/controllers/token.go 
line 226